routers/web/repo/setting_protected_branch.go

```diff
@@ -115,21 +117,20 @@ func SettingsProtectedBranch(c *context.Context) {
 	c.Data["whitelist_users"] = strings.Join(base.Int64sToStrings(rule.WhitelistUserIDs), ",")
 	c.Data["merge_whitelist_users"] = strings.Join(base.Int64sToStrings(rule.MergeWhitelistUserIDs), ",")
 	c.Data["approvals_whitelist_users"] = strings.Join(base.Int64sToStrings(rule.ApprovalsWhitelistUserIDs), ",")
+	c.Data["status_check_contexts"] = strings.Join(rule.StatusCheckContexts, "\n")
 	contexts, _ := git_model.FindRepoRecentCommitStatusContexts(c, c.Repo.Repository.ID, 7*24*time.Hour) // Find last week status check contexts
-	for _, ctx := range rule.StatusCheckContexts {
-		var found bool
-		for i := range contexts {
-			if contexts[i] == ctx {
-				found = true
-				break
+	c.Data["branch_status_check_contexts"] = contexts
```

Comment:
`branch_status_check_contexts` it needs a new name I guess. `recent_status_check_records` ?

Reply:
Is `recent_status_checks` ok?

Reply:
Sure, as long as it's better than before.

Reply:
done in 087061d9b330b1bc2b126d7028fcb06b799fba14